.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,85 @@
+name: e2e
+
+on: [pull_request]
+
+jobs:
+  privilege-escalation:
+    name: Privilege escalation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install curl
+        run: |
+          sudo apt -y update
+          sudo apt -y install curl
+
+      - name: Download and install minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+          sudo dpkg -i ./minikube_latest_amd64.deb
+
+      - name: Start minikube and wait until minikube's CoreDNS is available
+        run: |
+          minikube start --driver=docker
+          kubectl wait --for=condition=available deployment coredns -n kube-system
+
+      - name: Build SRO
+        run: minikube image build -t sro:local .
+
+      - name: Install kubectl
+        uses: azure/setup-kubectl@v2.0
+
+      - name: Install NFD
+        run: kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.10.1
```

Comment:
What is the reason to not use https://github.com/kubernetes-sigs/node-feature-discovery/blob/87aca5d9974981e5d0874de339dd1971fd2fc4ac/Makefile#L116?

Same logic as before, it will be better IMO to not update the job each time the build process of NFD changes.

Reply:
This forces us to clone the repo instead of just one `kubectl` command. It is the [recommended quick installation instruction](https://github.com/kubernetes-sigs/node-feature-discovery#quick-start--the-short-short-version).
If your concern is that we never update the tag in the URL, we could just drop it and install `master`.

Reply:
Ack for using the deployment method from the documented "quick start".
Regarding the version of NFD I think it is better to not use `master`, we don't want some breaking jobs because of NFD's bugs.

Reply:
I agree, but what should be use then?

Reply:
Looks like `10.0.1` is the latest one at https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-nfd/images.yaml so let's keep it